main.go

```diff
@@ -92,6 +92,7 @@ var defaultWatchRepositoriesSync = []string{
 	"devicemonitor",
 	// client
 	"mender-binary-delta",
+	"monitor-client",
```

Comment:
I hereby request a comment preceding this line:
```c
// Monitoring
```

Reply:
ok